resources/access.rb

```diff
@@ -24,18 +24,18 @@
 property :access_user,   String, required: true, default: 'postgres'
 property :access_addr,   [String, nil], default: nil
 property :access_method, String, required: true, default: 'ident'
-property :notification,  Symbol, required: true, default: :reload
 
 action :grant do
+  config_resource = new_resource
```

Comment:
What does this bit do? Just provide a sensible alias?

Reply:
So this has to do with context of execution because the `new_resource` yielded in the `edit_resource` block is not the same as the `new_resource` provided by the config resource therefore I memoize it so I can signal the parent resource.

So for example, if I attach a notification to a `postgresql_config` resource and I use the `new_resource` given inside the `edit_resource` block, it won't trigger any notifications until much later because it will have a different copy of the resource versus the original definition of `postgresql_config`.